cmd/fission-bundle/main.go

```diff
@@ -200,7 +197,8 @@ Usage:
   fission-bundle --routerPort=<port> [--executorUrl=<url>]
   fission-bundle --executorPort=<port> [--namespace=<namespace>] [--fission-namespace=<namespace>]
   fission-bundle --kubewatcher [--routerUrl=<url>]
-  fission-bundle --storageServicePort=<port> --filePath=<filePath>
+  fission-bundle --storageServicePort=<port>
```

Comment:
Would we not need always two arguments - the port and type. And inside the Storage Svc class, the type will be used to activate differently. Also, it keeps arguments for StorageService consistent irrespective of the type.

Reply:
This is implemented now. Please verify.